build/resources/main/application-prod.yml

```
@@ -0,0 +1,49 @@
+swagger:
+  server:
+    url: ${SWAGGER_SERVER_URL:https://api.yourdomain.com}
+    description: 운영 서버
+
+# 운영환경 로깅 (ERROR 레벨만)
+logging:
+  level:
+    org.hibernate.SQL: warn
+    org.hibernate.type.descriptor.sql.BasicBinder: warn
+    com.example.login: info
+    root: warn
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)